common/src/Microsoft.Azure.IIoT.Hub.Module.Client/src/Default/IoTSdkFactory.cs

```diff
@@ -274,15 +277,27 @@ public sealed class ModuleClientAdapter : IClient {
                 if (IsClosed) {
                     return;
                 }
-                await _client.SendEventAsync(message);
+
+                try {
+                    await _client.SendEventAsync(message);
+                }
+                catch (Exception e) {
+                    _logger.Error(e, "Error sending message to IoT Hub");
+                }
             }
 
             /// <inheritdoc />
             public async Task SendEventBatchAsync(IEnumerable<Message> messages) {
                 if (IsClosed) {
                     return;
                 }
-                await _client.SendEventBatchAsync(messages);
+
+                try {
+                    await _client.SendEventBatchAsync(messages);
```

Comment:
Please undo - we should not eat exceptions - that should be done by any caller.

Reply:
Done